lib/post.php

```diff
@@ -83,6 +83,36 @@ function name() {
   }
 
   /**
+   * Returns the post_content
+   *
+   * Markdownify's the content if applicable
+   */
+  function content() {
+    $content = $this->post->post_content;
+
+    if ( function_exists( 'wpmarkdown_html_to_markdown' ) ) {
+      $content = wpmarkdown_html_to_markdown( $content );
+    }
+
+    return apply_filters( 'wpghs_content', $content );
+  }
+
+  /**
+   * Returns the file extension
+   *
+   * Use .md if markdown is enabled
+   */
+  function ext() {
+    $ext = ".html";
+
+    if ( function_exists( 'wpmarkdown_html_to_markdown' ) && function_exists( 'wpmarkdown_markdown_to_html' ) ) {
+      $ext = ".md";
+    }
+
+    return $ext;
+  }
+
+  /**
    * Calculates the proper GitHub path for a given post
```

Comment:
What would happen if I have a repo that was using HTML and then activated the plugin or vise versa? Wouldn't that break the entire site? Any smart way around that?